### PR TITLE
PCX-82: Data Cloud Studio additional instructions

### PR DESCRIPTION
**Pull request by nayaabahsan**
Addresses https://jira.cfops.it/browse/PCX-82 

Adding additional instructions on
- adding fields in Cloud Function (with note on debugging)
- adding fields in BigQuery
- updating fields and adding custom fields in Data Studio

### CODE REVIEW

File: products/logs/src/content/analytics-integrations/google-cloud/index.md [hunk 1]
```diff
@@ -220,6 +256,34 @@ To create a report for your log data based on the Cloudflare template:
 
 After you've added your report, you will notice that not all report components render successfully. To fix this, you need to resolve any errors related to invalid dimensions, metrics, or filters that appear in the affected report components.
 
+#### Update Data Studio with new fields
+
+To update Data Studio with fields added to BigQuery, refresh fields for the data source.
+
+1. Open the Data Studio Cloudflare Dashboard in **Edit** mode.
```
Comment (pedrosousa) [inline products/logs/src/content/analytics-integrations/google-cloud/index.md]: "Data Studio Cloudflare Dashboard" seems like a giant product name (is it?). Consider changing to "In Data Studio, open the Cloudflare Dashboard in **Edit** mode."

File: products/logs/src/content/analytics-integrations/google-cloud/index.md [hunk 2]
```diff
@@ -220,6 +256,34 @@ To create a report for your log data based on the Cloudflare template:
...
+
+2. Expand the **Resource** menu and select **Manage added data sources**.
+
+3. Click the **EDIT** action for the data source that you want to update.
+
+4. Click **REFRESH FIELDS** below the table. A window with Field changes found in BigQuery will pop up.
```
Comment (pedrosousa) [inline products/logs/src/content/analytics-integrations/google-cloud/index.md]: Should "Field changes found in BigQuery" be in **bold**?

File: products/logs/src/content/analytics-integrations/google-cloud/index.md [hunk 3]
```diff
@@ -220,6 +256,34 @@ To create a report for your log data based on the Cloudflare template:
...
+
+5. To add the new fields, click **APPLY**.
+
+You can also create custom fields directly in Data Studio:
+
+1. Open the Data Studio Cloudflare Dashboard in **Edit** mode.
```
Comment (pedrosousa) [inline products/logs/src/content/analytics-integrations/google-cloud/index.md]: Same comment on the "big product name".

File: products/logs/src/content/analytics-integrations/google-cloud/index.md [hunk 4]
```diff
@@ -220,6 +256,34 @@ To create a report for your log data based on the Cloudflare template:
...
+
+2. Expand the **Resource** menu and select **Manage added data sources**.
+
+3. Click the **EDIT** action for the data source that you want to add a custom field to.
+
+4. Click **ADD A FIELD** above the table.
+
+5. Enter a formula in the Formula editor.
```
Comment (pedrosousa) [inline products/logs/src/content/analytics-integrations/google-cloud/index.md]: Not sure if "Formula editor" should be in **bold**.

File: products/logs/src/content/analytics-integrations/google-cloud/index.md [hunk 5]
```diff
@@ -220,6 +256,34 @@ To create a report for your log data based on the Cloudflare template:
...
+
+2. Expand the **Resource** menu and select **Manage added data sources**.
+
+3. Click the **EDIT** action for the data source that you want to update.
+
+4. Click **REFRESH FIELDS** below the table. A window with Field changes found in BigQuery will pop up.
+
+5. To add the new fields, click **APPLY**.
+
+You can also create custom fields directly in Data Studio:
```
Comment (pedrosousa) [inline products/logs/src/content/analytics-integrations/google-cloud/index.md]: This is the only introductory sentence that ends with a colon. Should it end with a period, or should the other ones also end with a colon?

**Review by pedrosousa — APPROVED** [review]

File: products/logs/src/content/analytics-integrations/google-cloud/index.md [hunk 6]
```diff
@@ -76,6 +76,42 @@ Also, verify that the data now appears in your table in **BigQuery** by navigati
...
+
+To debug in Cloud Function, click **VIEW LOGS** on the Function details page. This will take you to the Logs Viewer, where any errors will appear.
+
+### Add fields in BigQuery
+
+To add fields in BigQuery, edit the schema.
+
+1. Open BigQuery.
+
+2. In the menu, expand **cloudflare-se-analytics**.
```
Comment (nayaabahsan) [inline products/logs/src/content/analytics-integrations/google-cloud/index.md]: the table name can be different for the customer, "your project name"

File: products/logs/src/content/analytics-integrations/google-cloud/index.md [hunk 7]
```diff
@@ -76,6 +76,42 @@ Also, verify that the data now appears in your table in **BigQuery** by navigati
...
+
+3. Expand **cloudflare_data** and click **cf_analytics_logs**.
```
Comment (nayaabahsan) [inline products/logs/src/content/analytics-integrations/google-cloud/index.md:105]: second name can be different; default name is cf_analytics_logs

File: products/logs/src/content/analytics-integrations/google-cloud/index.md (hunk 4 above)
Comment (nayaabahsan) [reply products/logs/src/content/analytics-integrations/google-cloud/index.md]: note to self - bold "Formula"

**Pull request merged**